### PR TITLE
Created WebAPI project in solution

### PR DESCRIPTION
**Pull request by abhijeetkumarLnG**
Added a WebAPI project in solution. Please review it.

### CODE REVIEW

File: Pspcl/Pspcl.API/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace Pspcl.API.Controllers
+{
+    [ApiController]
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.API/Controllers/WeatherForecastController.cs]: remove this default controller.

File: Pspcl/Pspcl.API/WeatherForecast.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+namespace Pspcl.API
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.API/WeatherForecast.cs]: Remove this class.

File: Pspcl/Pspcl.Models/Pspcl.Models.csproj [hunk 3]
```diff
@@ -6,4 +6,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.Models/Pspcl.Models.csproj]: Why we have installed rest sharp in models?

File: Pspcl/Pspcl.Web/Controllers/HomeController.cs [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            var _client = new RestClient(_configuration["ApiUrl"]);
+            var request = new RestRequest("Weatherforecast", Method.Get);
+            var response= await _client.ExecuteGetAsync(request);
+            TempData.Keep("Message");
+            return View();
+        }
+        public IActionResult Privacy()
+
+        {
+            TempData["Message"] = "Temp Data Content";
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.Web/Controllers/HomeController.cs]: What is the use of this controller class?

File: Pspcl/Pspcl.Web/Program.cs [hunk 5]
```diff
@@ -19,9 +21,17 @@
 app.UseRouting();
 
 app.UseAuthorization();
+app.UseEndpoints(endpoints =>
+{
+    app.MapControllerRoute(
+    name: "default",
+    pattern: "{controller=Home}/{action=Index}/{id?}");
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.Web/Program.cs]: Why this custom controller route has been added?

**Review by VikasNischalDA — CHANGES_REQUESTED** [review]
Is swagger has been implemented in WebAPI?

File: Pspcl/Pspcl.API/Controllers/WeatherForecastController.cs (hunk 1 above)
Comment (abhijeetkumarLnG) [reply Pspcl/Pspcl.API/Controllers/WeatherForecastController.cs]: I removed it.

File: Pspcl/Pspcl.API/WeatherForecast.cs (hunk 2 above)
Comment (abhijeetkumarLnG) [reply Pspcl/Pspcl.API/WeatherForecast.cs]: I removed it.

File: Pspcl/Pspcl.Models/Pspcl.Models.csproj (hunk 3 above)
Comment (abhijeetkumarLnG) [reply Pspcl/Pspcl.Models/Pspcl.Models.csproj]: I uninstalled it from models

**Comment by abhijeetkumarLnG** [comment]
I will raise a new PR with the changes, so i am closing it.

**Pull request closed without merging**